__tests__/send.test.js

```diff
@@ -74,7 +74,7 @@ test('Should be able to send ETH on Ropsten', async (t) => {
   await t.expect(getAllByText(FIXTURE_SEND_CONTACT)).ok();
 
   // Send TX
-  await t.click(getByText(getTransValueByKey('CONFIRM_AND_SEND')));
+  await t.click(getByText(new RegExp(getTransValueByKey('CONFIRM_AND_SEND'), 'i')));
```

Comment:
its not immediately clear to me what the purpose of this change is.

Reply:
It seems that sometimes the text is rendered differently so we use a regex pattern to get the values like in the videos `getByText(/confirm and send/i)`.
Since the search value is a variable, the only way i found how to do dynamic regex patterns in JS is with `new RegExp()`